SU2_CFD/src/output_structure.cpp

```diff
@@ -4726,12 +4726,12 @@ void COutput::SetConvHistory_Body(ofstream *ConvHist_file,
           Total_IDC     = solver_container[val_iZone][FinestMesh][FLOW_SOL]->GetTotal_IDC();
         }
         
-        if (inv_design) {
+    /*  if (inv_design) {
```

Comment:
Everything looks good, other than a couple things that are commented, like this 'inv_design' section. Was this intended?

Reply:
Yes, because this variable is not used and leads to a compiler warning. This was merged in earlier, but apparently was reverted by another PR.

Reply:
I can remove it, if you prefer.